<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2249</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>2131</width>
      <height>841</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="verticalGroupBox">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="verticalGroupBox_2">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>      Select Play to Draft or calculate draft odds</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButtonDraftPlayer">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Draft Selected Player</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonUndoDraftedPlayer">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Undo Drafted Player</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>                             Search Player</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPlayerSearch">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetPlayerList">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Load fantasy data to start</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox_2">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="12">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="6">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="8">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="11">
           <widget class="QLineEdit" name="lineEditStartMockDraft">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="14">
           <widget class="QPushButton" name="pushButtonDownloadDraftData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Download Draft Data</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>     League Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxLeagueType">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Standard</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PPR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2 QB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dynasty</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dynasty Rookie</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="10">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="5">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>   Your Draft Position  </string>
            </property>
           </widget>
          </item>
          <item row="1" column="9">
           <widget class="QPushButton" name="pushButtonResetDraftData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Reset Data</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>  Number of Teams  </string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="comboBoxNumberOfTeams">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>8 Team</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10 Team</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12 Team</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14 Team</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QPushButton" name="pushButtonLoadFantasyData">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Load Draft Data</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QComboBox" name="comboBoxDraftPick">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>Pick 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick 8</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="13">
           <widget class="QLineEdit" name="lineEditMockDraftEnd">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="11">
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Mock Draft Start</string>
            </property>
           </widget>
          </item>
          <item row="0" column="13">
           <widget class="QLabel" name="label_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Mock Draft End</string>
            </property>
           </widget>
          </item>
          <item row="1" column="15">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidgetDraftTable">
          <property name="minimumSize">
           <size>
            <width>951</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalGroupBox_2">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>             Select player and round</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>                to find your odds</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonCalculateDraftOdds">
              <property name="text">
               <string>Calculate Draft Odds</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxCalculateOddsRound">
              <item>
               <property name="text">
                <string>Round 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 9</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 11</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 12</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 13</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 14</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Round 15</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserCalculateOdds"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2249</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
